### DISCUSSION TITLE
Scope highlight nearest?

### DISCUSSION THREAD

**Discussion by geril07** [lukas-reineke/indent-blankline.nvim#718] [Q&A]
Is it possible to highlight nearest scope and in some files there is no highlight(lua for example, they are only in function scope). Can i have always highlight nearest?
![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/62308020/47661b1c-aa64-4345-a7b4-61fff3c55100)
![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/62308020/8fd055e0-6e49-4299-a80d-db36d7b40ed1)

**Comment by TheTrueWhiteOwl** [comment]
A while ago some work was put into trying to implement this feature, but I think they are not gonna continue and finish the implementation (at least for now). See the following issue: https://github.com/lukas-reineke/indent-blankline.nvim/issues/649

I also want to point out that indent blankline currently has the feature of highlighting the nearest scope not indent level! That is why it highlights inside lua functions, but not inside lua tables. More info here: https://github.com/lukas-reineke/indent-blankline.nvim/issues/632
